### ISSUE TITLE
";chats" command unlinks %chat-cli on moons

### ISSUE THREAD

**Issue by KarelPeeters123** [urbit/urbit#2167]

**Comment by KarelPeeters123** [comment]
i get the following output when executing the command:
```
[unlinked from [p=~lantel-savwep-borlyt-salnus q=%chat-cli]]
/gx/~borlyt-salnus/chat-store/~2020.1.13..16.39.31..fa74/keys/noun
[%drum-coup-fail ~lantel-savwep-borlyt-salnus p=~lantel-savwep-borlyt-salnus q=%chat-cli]
```

and i get kicked from the chat prompt and put into my dojo and unable to return to the chat prompt with ctrl-x. executing `|link %chat-cli` fixes it again

**Comment by Fang-** [comment]
Yeah, this is known broken. It tries to scry into your planet's chat-store for the chats you're in (because it _subscribes_ to your planet for all messages), but remote scries are asynchronous and not supported in gall applications.

https://github.com/urbit/urbit/blob/2089045a4e1ed50cd13084b6cb00afabc7dbe0f6/pkg/arvo/app/chat-cli.hoon#L1019-L1023

We should probably open a subscription and keep local state on it instead?

**Comment by matildepark** [comment]
Closed via #2901.
